### PR TITLE
Fix data science pipeline tests in Smoke due to changes related to pipeline versions

### PR DESCRIPTION
**Pull request by jgarciao**
- Quick fixes for pipeline tests in smoke due to changes in 2.7. **I'll send a follow-up PR to enhance them**
- Adds DataSciencePipelines tag to all pipeline test suites
- Tested successfully in rhods-ci-pr-test/2452

### CODE REVIEW

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource [hunk 1]
```diff
@@ -133,10 +127,28 @@
...
+    ${pipeline_row_xp}=    Set Variable    ${PIPELINES_TABLE_XP}/tbody/tr/td/h2[text()="${pipeline_name}"]/../..
+    ${kebab_xp}=           Set Variable    ${pipeline_row_xp}/td/button[@aria-label="Kebab toggle"]
+    ${kebap_action_xp}=    Set Variable    ${pipeline_row_xp}/td/div/div/ul/li/button[.//*[contains(text(), '${action}')]]
+    Wait Until Page Contains Element    ${kebab_xp}
+    Click Element                       ${kebab_xp}
+    Wait Until Page Contains Element    ${kebap_action_xp}
+    Click Element                       ${kebap_action_xp}
+
+
+Click Action From Pipeline Run Actions Menu
```
[the remark this answers is not in the document]
Comment (jstourac) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource:140]: this one is cheap to fix

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource [hunk 2]
```diff
@@ -119,3 +125,9 @@ Delete Pipeline Server
         Sleep  1s
     END
     Should Be True    ${pod_count} == 0    Pods weren't removed
+
+Verify There Is No "Error Displaying Pipelines" After Creating Pipeline Server
+    [Documentation]    Verify me message "Error displaying Pipelines" after creating pipeline server
```
Comment (jstourac) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource:130]: `Verify me message` ?

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource [hunk 3]
```diff
@@ -119,3 +125,9 @@ Delete Pipeline Server
...
+    ...    Bug: RHOAIENG-2931
+    Run Keyword And Ignore Error       Wait Until Page Contains    Error displaying pipelines    timeout=60s
+    Run Keyword And Warn On Failure    Page Should Not Contain     Error displaying pipelines
```
Comment (jstourac) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource:133]: Just to be sure - at the beginning there is an `Error Displaying Pipelines` message and we give it 60 seconds to disappear. This is the intention, correct?

**Review by jstourac — COMMENTED** [review]
Just few minor comments. LGTM in general :slightly_smiling_face:

**Review by diegolovison — APPROVED** [review]

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource [hunk 4]
```diff
@@ -133,10 +127,28 @@ Create Pipeline Run    # robocop: disable
...
     [Documentation]    Click an action from Actions menu (3-dots menu on the right)
     [Arguments]    ${pipeline_name}    ${action}
-    Wait Until Page Contains Element    xpath=//tr[td[.//a[.//text()="${pipeline_name}"]]]/td[contains(@class,"-table__action")]//button[@aria-label="Kebab toggle"]    # robocop: disable
-    Click Element    xpath=//tr[td[.//a[.//text()="${pipeline_name}"]]]/td[contains(@class,"-table__action")]//button[@aria-label="Kebab toggle"]    # robocop: disable
-    Wait Until Page Contains Element    xpath=//tr[td[.//a[.//text()="${pipeline_name}"]]]//td//li//button[.//*[contains(text(), '${action}')]]    # robocop: disable
-    Click Element    xpath=//tr[td[.//a[.//text()="${pipeline_name}"]]]//td//li//button[.//*[contains(text(), '${action}')]]    # robocop: disable
+
+    ${pipeline_row_xp}=    Set Variable    ${PIPELINES_TABLE_XP}/tbody/tr/td/h2[text()="${pipeline_name}"]/../..
+    ${kebab_xp}=           Set Variable    ${pipeline_row_xp}/td/button[@aria-label="Kebab toggle"]
+    ${kebap_action_xp}=    Set Variable    ${pipeline_row_xp}/td/div/div/ul/li/button[.//*[contains(text(), '${action}')]]
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource:133]: this looks a very fragile xpath, easily to break

**Review by bdattoma — APPROVED** [review]

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource (hunk 3 above)
Comment (jgarciao) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource:133]: The "Error Displaying Pipelines" message can take a few secs to appear, so we wait for it for 60s. After that, if the error message is not there we do nothing  (bug is fixed) but if it's present we log a Warning (I was hesitating if to fail the test, maybe I'll do that in Sanity)

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource (hunk 4 above)
Comment (jgarciao) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource:133]: I agree. It should be revisited in a future PR

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource (hunk 2 above)
Comment (jgarciao) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource:130]: Thanks. I'll fix ti in a follow-up PR

**Pull request merged**